src/app/components/index/index.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { TokenService } from '../../services/token.service';
+
+@Component({
+  selector: 'app-index',
+  templateUrl: './index.component.html',
+  styleUrls: ['./index.component.css']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
  styleUrls: ["./index.component.css"]
```